packages/clerk-js/src/ui/utils/createSlug.ts

```diff
@@ -0,0 +1,2089 @@
+const escapeStringRegexp = (string: string): string =>
+  string.replace(/[|\\{}()[\]^$+*?.]/g, '\\$&').replace(/-/g, '\\x2d');
+
+const doCustomReplacements = (string: string, replacements: string[][]): string => {
+  for (const [key, value] of replacements) {
+    string = string.replace(new RegExp(escapeStringRegexp(key), 'g'), value);
+  }
+
+  return string;
+};
+
+export const transliterate = (string: string): string => {
```

Comment:
⛔  Team, transliteration should happen in one place, ideally FAPI in order to have the exact same result across all flows. So, we should replace this with a FAPI call to autofill slub on the frontend.

cc: @alex-ntousias